BoardGameTrainer/CustomWidgets/StopConditionFrame.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Game.IGame;
+
+namespace BoardGameTrainer
+{
+    public class StopConditionFrame : DropDownFrame<IStopConditionFactory>
+    {
+        private SpinButtonBox[] Boxes { get; } = new SpinButtonBox[2];
+        private int VisibleBox { get; set; } = 0;
+        public int Param { get; set; } = 1000;
+
+        public event EventHandler ParamChanged
+        {
+            add { Boxes[0].Changed += value; Boxes[1].Changed += value; DropDown.Changed += value; }
+            remove { Boxes[0].Changed -= value; Boxes[1].Changed -= value; DropDown.Changed -= value; }
+        }
+
+        public StopConditionFrame(Dictionary<string, IStopConditionFactory> entries) : base("Stop condition", entries)
+        {
+
+            Boxes[0] = new(1000, 100000, 100, Param, "miliseconds");
```

Comment:
Czemu nie możemy zejść poniżej 1000 ms? to dość długi czas
Dałabym 100ms jako dolne ograniczenie